graspologic/match/qap.py

```diff
@@ -455,13 +462,23 @@ def _quadratic_assignment_faq(
 
     # [1] Algorithm 1 Line 2 - loop while stopping criteria not met
     for n_iter in range(1, maxiter + 1):
+        if verbose==1:
+            print("Running Frank-Wolfe iterations")
+        elif verbose>1:
+            print(f"Iteration {n_iter}")
```

Comment:
```suggestion
        if verbose>=1:
            print(f"Running Frank-Wolfe iteration {n_iter}")
```
we can probably just compress these two